static/css/uqcs.css

```diff
@@ -1,7 +1,7 @@
 /*UQCS Website Style Sheet*/
 
 .uqcs-background {
-    background-image: url(../img/hero-background.png);
+    background-image: url(..{{ .Site.BaseURL }}/img/hero-background.png);
```

Comment:
The double .. won't work as a prefix.

Reply:
This is the danger of find replace at 5:30 on a friday.